include/uv.h

```diff
@@ -1414,9 +1414,19 @@ UV_EXTERN void uv_key_delete(uv_key_t* key);
 UV_EXTERN void* uv_key_get(uv_key_t* key);
 UV_EXTERN void uv_key_set(uv_key_t* key, void* value);
 
+
 typedef void (*uv_thread_cb)(void* arg);
 
 UV_EXTERN int uv_thread_create(uv_thread_t* tid, uv_thread_cb entry, void* arg);
+UV_EXTERN int uv_thread_cpumasksize(void);
```

Comment:
If `uv_thread_cpumasksize()` never returns a value < 0, it's better to make it `size_t` or `unsigned`.